runtime/services/ttsd/index.js

```diff
@@ -78,6 +79,8 @@ function reConnect (CONFIG) {
     _TTS.reconnect()
     return
   }
+  // for detail, see https://developer.rokid.com/docs/3-ApiReference/openvoice-api.html
```

Comment:
Use the link https://developer.rokid.com/docs/3-ApiReference/openvoice-api.html#ttsrequest?

Reply:
this url is document of speech/tts protocol.
it explain what is the 'declaimer'